recipe/set_bld_opts.bat

```diff
@@ -62,7 +62,7 @@ set BLD_OPTS=%WIN64% ^
     NETCDF_LIB=%LIBRARY_LIB%\netcdf.lib ^
     NETCDF_SETTING=yes ^
     NETCDF_HAS_NC4=yes ^
-    OPENJPEG_CFLAGS="-I%LIBRARY_INC%\openjpeg-2.4" ^
+    OPENJPEG_CFLAGS="-I%LIBRARY_INC%\openjpeg-2.5" ^
```

Comment:
```suggestion
    OPENJPEG_CFLAGS="-I%LIBRARY_INC%\openjpeg-%openjpeg%" ^
```
should work. I believe pinnings are exported as environment variables. I would test.

Reply:
Great suggestion!  Let's give it a try!